draft-ietf-mls-extensions.md

```diff
@@ -881,14 +882,13 @@ described in {{safe-hpke}} with an empty context. Other functions are defined in
 The sender MUST set the authentication scheme to
 `TargetedMessageAuthScheme.HPKEAuthPsk`.
 
-As described in {{safe-hpke}} the `hpke_context` is a LabeledExtensionContent struct
+As described in {{safe-hpke}} the `hpke_context` is an ExtensionContext struct
```

Comment:
I'm not sure what an `ExtensionContext` struct is. Is that defined anywhere?

I think this whole section doesn't make sense anymore. The HPKE section has changed, but also seems incomplete.

Reply:
@raphaelrobert Are you saying the HPKE section seems incomplete?  Because it only covers the Base mode?  Seems like a discussion for another issue.

I agree that this section needs to either get removed or be refactored to refer to the safe HPKE interface as it is now defined.  I would probably argue for the former, since Auth/AuthPSK modes don't work with any PQ KEMs.

Reply:
I hadn't realized the auth mode wasn't exposed. I'm fine dropping it, it would streamline targeted messages. 
The fact that the AuthPSK mode is not exposed is however problematic. It was used for channel binding, so that targeted messages required the recipient to be a full group member and not just the owner of the private key that corresponds to HPKE public key of a leaf.

For the sake of progress, I propose to revert the change above and do a separate PR that aligns targeted messages with the safe API, provided we find a solution to the PSK injection.